src/core/src/main/java/org/apache/jmeter/testelement/TestPlan.java

```diff
@@ -83,11 +86,23 @@ public void prepareForPreCompile()
 
     /**
      * Fetches the functional mode property
-     *
+     * Could be change for no gui test with jmeter property : PROP_FUNCTIONAL_MODE
```

Comment:
*Can be changed for non-gui test with JMeter property: {@code PROP_FUNCTIONAL_MODE}*

Reply:
Changed with proposal modification